pkg/tfbridge/provider.go

```diff
@@ -1048,6 +1049,11 @@ func (p *Provider) Delete(ctx context.Context, req *pulumirpc.DeleteRequest) (*p
 	return &pbempty.Empty{}, nil
 }
 
+// Construct creates a new instance of the provided component resource and returns its state.
+func (p *Provider) Construct(context.Context, *pulumirpc.ConstructRequest) (*pulumirpc.ConstructResponse, error) {
+	return &pulumirpc.ConstructResponse{}, errors.New("resources may not be constructed")
```

Comment:
I think you should do something like:

```
return nil, status.Error(codes.Unimplemented, "Construct is not yet implemented")
```

rather than throw an error - thoughts?

Reply:
Ah, yes.